### ISSUE TITLE
test: switch the algorithm and the distance metric for sklearn.neighbors.KNeighborsClassifier

### ISSUE THREAD

**Issue by iomz** [Interactions-HSG/MyoKTROS#37]
In addition to Euclidean Distance, make other `metrics` available. The default is "minkowski", but "eucredean" is currenlty used.
- https://scikit-learn.org/stable/modules/generated/sklearn.metrics.pairwise.distance_metrics.html#sklearn.metrics.pairwise.distance_metrics

The same for `algorithm`, but my guess is "auto" is just right...
